test/evergreen/coverage-report.sh

```diff
@@ -16,6 +16,8 @@ python_binary=$2
 github_commit=$3
 pr_args=$4
 
+echo "Current args: $@"
```

Comment:
Hi @bzm-jas , printing the arguments will be valuable, but I think the best way to do that is to print each variable out (eg is_patch, python_binary etc) so we can easily confirm that the right values have ended up in the right variables.

Reply:
But obviously we can print $@ too.

Reply:
Hi Jeremy, I've added more print info in the most recent commit: [d10d18c](https://github.com/wiredtiger/wiredtiger/pull/10952/commits/d10d18c42126575e18b2fa97a6af0b2d304c1d9e)